examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -125,51 +128,118 @@ bool OTAProviderExample::SelectOTACandidate(const uint16_t requestorVendorID, co
     return candidateFound;
 }
 
+UserConsentSubject OTAProviderExample::GetUserConsentSubject(const app::CommandHandler * commandObj,
+                                                             const app::ConcreteCommandPath & commandPath,
+                                                             const QueryImage::DecodableType & commandData, uint32_t targetVersion)
+{
+    UserConsentSubject subject;
+    FabricIndex fabricIndex = commandObj->GetSubjectDescriptor().fabricIndex;
+    subject.fabricId        = Server::GetInstance().GetFabricTable().FindFabricWithIndex(fabricIndex)->GetFabricId();
+    subject.nodeId          = commandObj->GetSubjectDescriptor().subject;
+    subject.endpointId      = commandPath.mEndpointId;
```

Comment:
Which endpoint is it? The local endpoint ID. Suggest renaming as `providerEndpointId`

Reply:
Changed to `providerEndpointId`